include/rviz_visual_tools/remote_control.hpp

```diff
@@ -112,18 +114,17 @@ class RemoteControl
   }
 
 private:
-  // A shared node handle
-  ros::NodeHandle nh_;
+  // Node Interfaces
+  rclcpp::node_interfaces::NodeTopicsInterface::SharedPtr topics_interface_;
+  rclcpp::Logger logger_;
 
   // Short name for this class
   std::string name_ = "remote_control";
 
   // Input
-  ros::Subscriber rviz_dashboard_sub_;
+  rclcpp::Subscription<sensor_msgs::msg::Joy>::SharedPtr rviz_dashboard_sub_;
 
-  // Debug interface
-  bool is_waiting_ = false;
-  bool next_step_ready_ = false;
+  std::shared_ptr<std::promise<bool>> next_step_ready_;
```

Comment:
do we really need a shared_ptr here?? re:@tylerjw's debates, seems like excess

Reply:
changed to unique